lib/ex_doc/formatter/html.ex

```diff
@@ -390,6 +385,35 @@ defmodule ExDoc.Formatter.HTML do
     |> Enum.sort_by(fn extra -> GroupMatcher.group_index(groups, extra.group) end)
   end
 
+  def generate_redirects(config, ext) do
+    config.redirects
+    |> Map.put_new("index", config.main)
+    |> Enum.map(fn {from, to} ->
+      source = stringify_redirect_item(from) <> ext
+      destination = stringify_redirect_item(to) <> ext
+      generate_redirect(source, config, destination)
+
+      source
+    end)
+  end
+
+  defp stringify_redirect_item(item) when is_binary(item) do
+    item
+  end
+
+  defp stringify_redirect_item(item) when is_atom(item) do
+    inspected = inspect(item)
+
+    case to_string(item) do
```

Comment:
Is there a better way to inspect an atom that will leave off both `:` and `Elixir.`?

Reply:
Or should we just treat any atoms as module names? and do `inspect(item)`. I was just getting ahead of folks writing:

```elixir
redirects: %{
  :upgrade => :quickstart,
  Old.Module => New.Module
}
```

Reply:
There is likely already a function in this project that does that, but it may be specific for each supported language. In that case, our options are:

1. Do not support atoms
2. Support atoms but simply call `to_string` on it and yank the "Elixir." prefix

Reply:
👍 that was the other alternative I considered, but thought there could be pages that start with `Elixir.`. So perhaps just only supporting strings is the better answer?

Reply:
It no longer supports atoms.